src/ui_framework/doc_site/src/components/guide_example/guide_example.jsx

```diff
@@ -0,0 +1,77 @@
+
+import React, {
+  Component,
+  PropTypes,
+} from 'react';
+
+import {
+  Slugify,
+} from '../../services';
+
+import {
+  GuidePage,
+  GuidePageSection,
+} from '../';
+
+export default class GuideExample extends Component {
+
+  constructor(props, sections) {
```

Comment:
I think you have a really interesting point, but I'd like to avoid too much re-architecting of how the docs site works within this PR -- I just want to focus on getting it integrated with Kibana for now, so we can unblock UI development. I'm totally open to refactoring it later though. :)